README.adoc

```diff
@@ -1791,6 +1791,26 @@ class MyControllerTest < ActionDispatch::IntegrationTest
 end
 ----
 
+== Additional Testing Resources [[additional-testing-resources]]
```

Comment:
I think this could just be part of the previous 'Testing' section?

Reply:
Oh, I see. Yeah, I think this is better, there's not really any convention of following the Rails Guides structure.